concepticondata/references/references.bib

```diff
@@ -4095,3 +4095,13 @@ @Article{Daniel2021
   Doi                                 = {doi:10.1353/lan.2021.0046},
   Number                              = {3}
 }
+
+@Article{Oskolskaya2021,
+  doi                                 = {10.1075/dia.20010.osk},
+  url                                 = {https://doi.org/10.1075%2Fdia.20010.osk},
+  year                                = 2021,
```

Comment:
The "{}" are missing.

Reply:
Bibtex has the bad property to accept missing {} if there is no whitespace in between, but it is more consisten to always add them, as some parsers do not accept this.